### DISCUSSION TITLE
Dugong Railgun Shell Rack???

### DISCUSSION THREAD

**Discussion by MarkCringan** [FakeFishGames/Barotrauma#10009] [Ideas and suggestions]
Dugong still has a railgun shell rack in armory despite not being capable of equipping a railgun.

**Comment by Steamlined** [comment]
Both Orcas have the same thing, I've noticed

**Comment by bookslayer10** [comment]
It doesn't make a lot of sense, but it does lets you store looted railgun shells from other ships, for when you want to switch to a different sub that actually has railguns. Also, pretty sure the Azimuth has an extra rack like this too.

**Reply by Remodor** [reply]
Yes, the Azimuth has one. Came here to report this.

**Comment by NotWendy** [comment]
Those racks were removed in our T1 submarine reworks in this year's spring update. :)
